tests/helpers/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "test_helpers"
+version = "0.0.0"
+edition = "2018"
+authors = ["Rafael Rivera", "Microsoft"]
```

Comment:
Unpublished packages don't need authors. I'd also like to avoid keeping track of all the contributors in code. GitHub does a better job of highlighting contributors.  
```suggestion
```